library/tests/fy_py_files/test_fy_py_files/property_with_generics_impl/abc_fy.py

```diff
@@ -0,0 +1,21 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+#  file, You can obtain one at https://mozilla.org/MPL/2.0/.
+"""fy
+from .greetings_t import SpanishGreeting
+
+
+property spanish_greeting: SpanishGreeting
```

Comment:
```suggestion
property greeting[GreetingT]:
```

This class needs to be generic. That is how it is used in the flow.
property greeting[SpanishGreeting]